frontend/src/components/Checkbox/index.js

```diff
@@ -0,0 +1,29 @@
+import { PropTypes } from "prop-types";
+
+import styles from "./index.module.css";
+
+function Checkbox({ id, children }) {
+  return (
+    <>
+      <input id={id} className={styles.input} type="checkbox"></input>
+      <label htmlFor={id}>{children}</label>
+      <span className={styles.control}>
+        <svg viewBox="0 0 24 24" aria-hidden="true" focusable="false">
+          <path
+            fill="none"
+            stroke="currentColor"
+            strokeWidth="4"
+            d="M1.73 12.91l6.37 6.37L22.79 4.59"
+          />
+        </svg>
+      </span>
+    </>
+  );
+}
+
+Checkbox.propTypes = {
+  id: PropTypes.string.isRequired,
+  children: PropTypes.string.isRequired,
```

Comment:
You're missing a `disabled` prop. Keep in mind that you can set default states as well. 

https://reactjs.org/docs/typechecking-with-proptypes.html

Reply:
I don't understand what you mean with disable prop :/

Reply:
All input elements can also be `disabled` > [Check](https://www.geeksforgeeks.org/html-dom-input-checkbox-disabled-property/#:~:text=A%20disabled%20checkbox%20is%20unclickable,the%20Input%20Checkbox%20disabled%20property.)

Reply:
You can also check out Ricardo's `input` element.

Reply:
Will do!